notebooks/02_model/mmlspark_lightgbm_criteo.ipynb

```diff
@@ -0,0 +1,449 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>Copyright (c) Microsoft Corporation. All rights reserved.</i>\n",
+    "\n",
+    "<i>Licensed under the MIT License.</i>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Content Based Personalization\n",
+    "## LightGBM on Azure Databricks<br>\n",
+    "This notebook provides a quick example of how to train LightGBM model on Azure Databricks and deploy it using MML Spark for a content personalization scenario.<br><br>\n",
+    "[LightGBM](https://github.com/Microsoft/Lightgbm) \\[1\\] is a gradient boosting framework that uses tree-based learning algorithms.<br>\n",
+    "[MMLSpark](https://github.com/Azure/mmlspark) \\[2\\] allows LightGBM to be called in a Spark environment which provides several advantages:\n",
+    "- Distributed computation for model development\n",
+    "- Easy integration into existing Spark workflows\n",
+    "- Model serving through Spark Serving \\[3\\]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Global Settings and Imports"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A python script is provided to simplify setting up Azure Databricks with the correct\n",
+    "dependencies.<br> Run ```python scripts/databricks_install.py -h``` for more details."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "System version: 3.6.7 | packaged by conda-forge | (default, Nov 21 2018, 03:09:43) \n",
+      "[GCC 7.3.0]\n",
+      "PySpark version: 2.3.1\n",
+      "MMLSpark version: Azure:mmlspark:0.16\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "\n",
+    "sys.path.append(\"../../\")\n",
+    "\n",
+    "import pyspark\n",
+    "from pyspark.ml import PipelineModel\n",
+    "from pyspark.ml.feature import FeatureHasher\n",
+    "import papermill as pm\n",
+    "\n",
+    "from reco_utils.common.spark_utils import start_or_get_spark\n",
+    "from reco_utils.common.notebook_utils import is_databricks\n",
+    "from reco_utils.dataset.criteo import load_spark_df\n",
+    "from reco_utils.dataset.spark_splitters import spark_random_split\n",
+    "from scripts.databricks_install import MMLSPARK_INFO\n",
```

Comment:
This line is failing in databricks, it looks we are not uploading the scripts folders when we create the egg to DB. Once you change this to MMLSPARK_INFO = {"maven": {"coordinates": "Azure:mmlspark:0.16"}}, everything works. Will move the import inside the if

Reply:
solved